src/components/common/PageHeaderWrapper.tsx

```diff
@@ -97,7 +97,7 @@ export default function PageHeaderWrapper({ children, nextElement }: Props): JSX
   }, [debouncedSticky, debouncedScrollDown, height]);
 
   const styles: Record<string, any> = {
-    background: debouncedSticky ? theme.palette.TwClrBg : undefined,
+    background: debouncedSticky ? '#FBF9F9' : undefined,
```

Comment:
if we don't have a ticket to follow up on this hardcoded color, we should probably create one - just so we don't miss it?

Reply:
Good call, I will create one

Reply:
https://terraformation.atlassian.net/browse/SW-2202